components/DaoMenu.tsx

```diff
@@ -6,18 +6,18 @@ import { Menu, Transition } from '@headlessui/react';
 import { ChevronDownIcon } from '@heroicons/react/20/solid';
 import { useConfig } from '@/hooks/useConfig';
 
-type Option<T> = {
-  value: T;
+type Option = {
+  daoId: string;
```

Comment:
Wouldn't we want this to be the Dao enum instead of the daoId string

Reply:
sounds good to me, changed!